### DISCUSSION TITLE
Can't seem to switch Teams in development mode?

### DISCUSSION THREAD

**Discussion by infinite-persistence** [outline/outline#7901] [General] [answered]
1. In development mode
2. Created team using the `server/scripts/seed.ts` method using `email_1` (IP Wiki)
3. Invited `email_2` as a member.
4. Tweaked `seed.ts` to allow multiple teams (`if (teamCount < 10) {`) and seeded with `email_2` (SJ Wiki)).
    - :warning: This part I'm not sure if it's a valid method that mimics production.
6. From `email_2`, I can't seem switch teams from the `OrganizationMenu` -- clicking on IP Wiki below will always revert to my own SJ Wiki.

<img src="https://github.com/user-attachments/assets/83a48d3c-20f6-47e6-b5e7-8f2a0d9516c6" width="300">

**Comment by tommoor** [comment, marked as answer]
There is a forked codebase for cloud which adds support for multiple workspaces amongst other things – development mode of this repo has the same restrictions as the community edition (a single workspace).
